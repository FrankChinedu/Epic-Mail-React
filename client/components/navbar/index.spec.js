import React from 'react';
import { shallow } from 'enzyme';
import { NavbarComponent } from './index';

describe('Navbar', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<NavbarComponent debug />);

    expect(component).toMatchSnapshot();
  });
});
