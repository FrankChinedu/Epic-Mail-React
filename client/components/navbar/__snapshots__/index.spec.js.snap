// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar should render correctly in "debug" mode 1`] = `
<Fragment>
  <header
    className="nav-header"
  >
    <div
      className="flex justify-content-sb"
    >
      <Link
        to="/"
      >
        <h1
          className="brand-name flex"
        >
          <span
            className="black"
          >
            EPIC Mail
          </span>
          <span
            className="brand-image"
          />
        </h1>
      </Link>
      <div
        className="align-self"
      >
        <h3
          className="m-0  sign-in-out"
        />
      </div>
    </div>
  </header>
</Fragment>
`;
